_projects/katka.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Katka"
+---
+
+# Katka 
+
+## Release Orchestration
+
+With Katka we aim create more visibility on release processes, find out where
+bottlenecks are and automate manual tasks. Katka interacts with different tools
+and applications like CI tools (Jenkins), Source Code Management solutions 
+(Bitbucket), Ticketing systems (JIRA) and others. It allows you to model your
+entire release flow with all the steps and actions that are part of the process
+of getting a single change from ticket to production.
+
+## Integrations
+
+We want to make Katka as flexible as possible when it comes to the tools that
+are being used. At first we will of course create integrations with our primary
+tools and services but it should be easy to integrate with others like GitHub,
+Gitlab, Travis, etc.
+
+## Automation
+
+Release processes can be very painful, especially in highly regulated 
```

Comment:
I will leave the last part of the sentince out (like our own)

Reply:
Fixed